sysdig/common/client.go

```diff
@@ -36,10 +42,28 @@ type sysdigCommonClient struct {
 }
 
 func (client *sysdigCommonClient) doSysdigCommonRequest(method string, url string, payload io.Reader) (*http.Response, error) {
-	request, _ := http.NewRequest(method, url, payload)
-	request.Header.Set("Authorization", "Bearer "+client.SysdigAPIToken)
+
+	var IBMInstanceID string
```

Comment:
I would prefer to not have IBM platform specific code here.

Is a pretty good concern and I have to ask about if we need to support IBM platform in addition to vanilla Sysdig platform. I'm going to raise this concern internally.

On any case, we need to think how to keep this open to allow others to implement these kinds of personalization.